apps/console/src/features/tenants/components/dropdown/tenant-dropdown.tsx

```diff
@@ -549,8 +549,8 @@ const TenantDropdown: FunctionComponent<TenantDropdownInterface> = (props: Tenan
                                                                     // modal is implemented.
                                                                     // style={{display:'none'}}
                                                                     data-testid={ 
-                                                                        `tenant-dropdown-link-
-                                                                        ${ name.replace(" ", "-") }` 
+                                                                        `tenant-dropdown-link-${ 
+                                                                            name.replace(" ", "-") }`
```

Comment:
I personally think good ol' string concatenation would be better in terms of formatting and readability. WDYT?

eg:

```
data-testid={ 
        "tenant-dropdown-link-" + 
        name.replace(" ", "-") 
}
```